### PR TITLE
[Word] Add snippet that shows how to get various table formatting

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: samples/word/40-tables/manage-formatting.yaml [hunk 1]
```diff
@@ -0,0 +1,281 @@
+order: 2
+id: word-tables-manage-formatting
+name: Get the formatting of a table
```
Comment (AlexJerabek) [inline samples/word/40-tables/manage-formatting.yaml:3]: Consider renaming this "Table formatting".

File: samples/word/40-tables/manage-formatting.yaml [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+        $("#get-table-row-alignment").click(() => tryCatch(getTableRowAlignment));
+        $("#get-table-row-border").click(() => tryCatch(getTableRowBorder));
+        $("#get-table-row-cell-padding").click(() => tryCatch(getTableRowCellPadding));
+        $("#get-table-cell-alignment").click(() => tryCatch(getTableCellAlignment));
+        $("#get-table-cell-border").click(() => tryCatch(getTableCellBorder));
+        $("#get-table-cell-cell-padding").click(() => tryCatch(getTableCellCellPadding));
+        $("#setup").click(() => tryCatch(insertTable));
+
+        async function getTableAlignment() {
+          // Get alignment details about the first table in the document.
```
Comment (AlexJerabek) [inline samples/word/40-tables/manage-formatting.yaml:23]: When you say "alignment details" here, is it in regards to the table itself, of the contents of the table? I would make similar clarifications for the other three alignment descriptions.

File: samples/word/40-tables/manage-formatting.yaml [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+            await context.sync();
+
+            console.log(`Details about the ${borderLocation} border of the first table:`);
+            console.log(`- Color: ${border.color}`);
+            console.log(`- Type: ${border.type}`);
+            console.log(`- Width: ${border.width} points`);
+          });
+        }
+
+        async function getTableCellPadding() {
```
Comment (AlexJerabek) [inline samples/word/40-tables/manage-formatting.yaml:52]: How does this differ from the row's cell padding?

**Review by AlexJerabek — CHANGES_REQUESTED** [review]
Thanks for merging these. I think the sample looks good overall. Maybe it's my lack of Word knowledge, but I'd love more clarity about the distinctions between the table/row/cell formatting meanings.

File: samples/word/40-tables/manage-formatting.yaml (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/40-tables/manage-formatting.yaml:3]: done

File: samples/word/40-tables/manage-formatting.yaml (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/40-tables/manage-formatting.yaml:23]: For table, it's a combination - I explain with the logs. I did clarify the alignment description for the others

**Review by AlexJerabek — APPROVED** [review]
Thanks for making those changes. After re-reading, I think the differences between table/row/cell are clear enough.

**Comment by AlexJerabek** [comment]
Oh, don't forget to update the snippet mapping metadata

**Comment by ElizabethSamuel-MSFT** [comment]
> Oh, don't forget to update the snippet mapping metadata

Yeah, I didn't want to do it before, especially when I was dealing with 3 snippets. Merging would have been painful.

**Pull request merged**